physical/postgresql/postgresql_test.go

```diff
@@ -260,13 +260,23 @@ func testPostgresSQLLockRenewal(t *testing.T, ha physical.HABackend) {
 	// Cancel attempt after lock ttl + 1s so as not to block unit tests forever
```

Comment:
Can we move this comment to L270? I think it would make more sense to be closer to the select statement.

Reply:
Yup! Moved in 9b59162ae